src/realm/parser/query_builder.cpp

```diff
@@ -885,4 +885,19 @@ void apply_ordering(DescriptorOrdering& ordering, ConstTableRef target, const pa
 }
 
 } // namespace query_builder
+
+
+// Function placed here in order not to require the Parser Library if this function is not used.
+Query Table::query(const std::string& query_string) const
```

Comment:
We might want to have an override that allows parsing in the mapping

Reply:
You got it